index.php

```diff
@@ -71,21 +73,44 @@
 				}
 				
 				
-				echo "<div id=\"commandIn\">
+				echo "<div id=\"container\">
```

Comment:
Maybe consider to put the html out of the PHP and use the PHP in the html.

the benefit of using `echo ""` with double quotes is that you can execute php variables. if you don't use any variable in your echo please use `echo 'your string';` this should give better performers.